lms/static/sass/_overrides.scss

```diff
@@ -11,13 +10,13 @@
   }
 }
 
-.header-global .logo {
+.global-header .logo {
   height: 50px;
   top: -10px;
 }
 
-.header-global,
-.header-global.slim {
+.global-header,
+.global-header.slim {
```

Comment:
Could you remove the [`width: 60% !important;` on line 29 below](https://github.com/BehavioralInsightsTeam/edx-theme/pull/2/files#diff-5aeea3d64f98c9950334a02e987400dcR29)?  That'll make the header sit right again.